docs/how-to/sandbox.md

```diff
@@ -22,8 +22,8 @@ from jina import Flow, Document
 f = Flow().add(uses='jinahub+sandbox://Hello')
 
 with f:
-  r = f.post('/', inputs=Document(text='world'), return_results=True)
-  print(r[0].text)
+    r = f.post('/', inputs=Document(text='world'), return_results=True)
+    print(r[0].text)
 ```
 
 This will start a Flow that only has one online Executor, and will send a document to it. The document will be processed by the Executor and the result will be returned.
```

Comment:
```suggestion
This will start a Flow that only has one online Executor, and will send a Document to it. The Document will be processed by the Executor and the result will be returned.
```